cvat/apps/engine/background.py

```diff
@@ -270,11 +251,44 @@ def _handle_rq_job_v1(
                     attachment_filename=filename,
                 )
 
-        if rq_job.is_finished:
+        action = self.request.query_params.get("action")
+
+        if action not in {None, "", "download"}:
+            raise serializers.ValidationError(
+                f"Unexpected action {action!r} specified for the request"
+            )
+
+        if not rq_job:
+            return None if action != "download" else \
+                HttpResponseBadRequest(
+                    "Result file cannot be send since there is no related job on the server. "
+                    "Initialize background process first without sending action=download parameter."
```

Comment:
```suggestion
                    "Unknown export request id. "
                    "Please request export first by sending a request without the action=download parameter."
```
I remember we had an agreement that users should not know anything about background processes.